01-algorithms/02-algorithms-divide-conquer.md

```diff
@@ -46,246 +44,163 @@ QuickSort is an algorithm which takes a divide-and-conquer approach to sorting a
 
-Larger merges would continue by comparing the values to be merged element by element, taking the smaller until one subarray is "empty." At that point, the remainder of the other array can be copied over!
+Let's solve this problem using the divide and conquer method.
 
-### !callout-info
+The first step is to break the problem down into subproblems of the same type. We can do so by splitting the list into two halves repeatedly until there is only one element to observe. We will also need to keep track of the minimum and maximum elements we have encountered thus far. We will update the min and max once we've reached the base case of one element in the list.
 
-## Copying and Merging Arrays
+1. Create placeholders for the minimum and maximum elements. Set the minimum to the maximum integer value, set the maximum to the minimum integer value.
+1. Base Case: If there's only one element in the list, update the min and max by doing two comparisons
+    1. If the min is bigger than the value of the one element in the list, update the min
+    1. If the max is smaller than the value of the one element in the list, update the max
+1. Recursively call the function on both halves of the list
+1. Return the min and max values
 
-How do we efficiently merge two arrays? An auxiliary array of size _n_ is often used to facilitate the merge steps, after which the values are copied from the auxiliary array back to the original array.
+In this algorithm we are dividing the list up until there is only one element, making comparisons of the minimum and maximum values to this one element and updating the values if needed, and then combining the results by storing the minimum and maximum values encountered during our recursion.
 
-### !end-callout
+![Performing a modified version of Merge Sort to identify the minimum and maximum values of an array](images/algorithmic-strategies_divide-and-conquer_min-max.png)  
+ _Fig. Performing a modified version of Merge Sort to identify the minimum and maximum values of an array._
 
-The two-way merging continues until there are no more subarrays and the original array is completely sorted.
+### !callout-info
 
-### Divide and Conquer Complexity in Merge Sort
+## Is there an echo in here?
 
-Since merge sort always divides the list in half at each divide step, there will be _log n_ levels in the division phase. By keeping data in place, and only adjusting indices to keep track of where the subarrays are located, at each level, there are at most _n_ calculations, for a time complexity of the division phase of _O(n log n)_.
+You may notice similarities between this algorithm and the Merge Sort algorithm described above. If so, that's great! The Merge Sort and Binary Search algorithms are excellent models for a large number of divide-and-conquer algorithms. Due to this fact, it is recommended to spend more time understanding Merge Sort and Binary Search than any other divide-and-conquer algorithms. 
```

Comment:
Love this! Consider modifying to make it clear understanding the algorithms is the goal, not memorization. 
```suggestion
You may notice similarities between this algorithm and the Merge Sort algorithm described above. If so, that's great! The Merge Sort and Binary Search algorithms are excellent models for a large number of divide-and-conquer algorithms. While it is not necessary to memorize these algorithms, we do recommend prioritizing thoroughly understanding how the Merge Sort and Binary Search algorithms work over any other divide-and-conquer algorithms. 
```